test/user.env.example

```diff
@@ -0,0 +1,10 @@
+export INSTANCE_NAME_PREFIX=colemick
+export LOCATION=westus2
```

Comment:
Is it better to use region instead of location since that's what Azure calls it?